rails-commenting.rb

```diff
@@ -48,15 +55,17 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
+      # ---8) This is the conditional for the RESTful route destroy. At this point in the method the user has been able to get the desired post with .find. Now they can destroy the data with the HTTP verb destroy which falls under D in CRUD. After they have succefully destroyed the data they are then redirected to the main page. 
       redirect_to blog_posts_path
     end
   end
 
   # ---9)
+  # This route known as private or strong params is used to help with security of the data sent through forms. Allowing the programmer to set which params are accepted. 
   private
   def blog_post_params
-    # ---10)
+    # ---10) 
+    # In the private params they are specifying what is permitted for all methods using strong params. Here they only want to allow the two symbols title and content. The point of this is to only allow sharing the needed information for user input
```

Comment:
Really great descriptions of what is happening in each of this methods.  You show a very good understanding of these Rails concepts.  Yay Spencer!